### PR TITLE
[CB-8618] Modified the medium duty template

### PR DESCRIPTION
**Pull request by atxmemo**
- change the idbroker node count to 1
- assign gateway roles on all nodes (including master)

### CODE REVIEW

File: core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha-722.bp [hunk 1]
```diff
@@ -186,6 +195,11 @@
             ],
             "refName": "solr-SOLR_SERVER-BASE",
             "roleType": "SOLR_SERVER"
+          },
+          {
+            "base": true,
+            "refName": "solr-GATEWAY-BASE",
```
Comment (orngejaket) [inline core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha-722.bp:201]: is there also not a hbase gateway and master base and a kafkabase and a hdfs base?  Or are they already there?

Comment (atxmemo) [reply core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha-722.bp:201]: looks like they are already there

Comment (orngejaket) [reply core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha-722.bp:201]: cools

File: core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha-722.bp [hunk 2]
```diff
@@ -20,7 +20,11 @@
           "kafka-GATEWAY-BASE",
           "kafka-KAFKA_BROKER-BASE",
           "solr-SOLR_SERVER-BASE",
-          "ranger-RANGER_ADMIN-BASE"
+          "ranger-RANGER_ADMIN-BASE",
+          "hdfs-GATEWAY-BASE",
+          "kafka-GATEWAY-BASE",
+          "hive-GATEWAY-BASE",
+          "hbase-GATEWAY-BASE"
```
Comment (kmanamcheri) [inline core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha-722.bp:27]: Why no solr-GATEWAY-BASE in the master node?

Comment (atxmemo) [reply core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha-722.bp:27]: missed it. good catch, https://github.com/hortonworks/cloudbreak/pull/8904

**Pull request merged**